RELEASE.md

```diff
@@ -0,0 +1,66 @@
+# How to create a new release
+
+This document describes how to create a new release of SBB Angular.
+
+## Preparation
+
+### Major release preparation
+
+- Update Angular packages to align with new angular major version.
+- Update ng-update schematics.
+- Update stackblitz assets.
+- Check TODOs, @breaking-change annotations and @deprecation annotations.
+- Check angular components commits to be considered for next major release on https://github.com/sbb-design-systems/sbb-angular/issues/1047.
+- Create new route for previous release (e.g. angular-v13.app.sbb.ch) and deploy corresponding version.
+- Github maintenance workflow
+  - Create new maintenance branch (e.g. 14.x).
+  - Create [branch protection rules](https://github.com/sbb-design-systems/sbb-angular/settings/branches) for the newly created branch.
+  - Edit TARGET_RELEASE in `.github/workflows/maintenance-tagging-workflow.yml`.
+  - Edit `baseBranches` property in `renovate.json` to activate renovate on new branch.
+
+## Release preparation
+
+The following steps create a release tag and generate a changelog:
+
+1. Check out the `master` or your desired release branch and ensure it's up-to-date.
+2. Run `yarn release` to create a tag and to generate the changelog.
+3. Check if the automatically updated `CHANGELOG.md` file contains all the information relevant for
+   the release. If everything worked well, continue with step 7.
+4. If necessary, update the `CHANGELOG.md` file. Afterwards, delete the tag created in step 3, e.g.
+   by running `git tag -d x.y.z`.
+5. Amend the modified `CHANGELOG.md` to the release commit by running
+   `git add CHANGELOG.md && git commit --amend`)
+6. Recreate the release tag: `git tag x.y.z`.
```

Comment:
Instead of changing the release commit, should we maybe just create a new `docs: adapt changelog` commit?
@mhaertwig, @jeripeierSBB Thoughts?

Reply:
but then this commit is after the tagged commit although it belongs somehow to the state of the tag in my opinion.

Reply:
That's fair.